### PR TITLE
fix: Hotfix to master

### PR DESCRIPTION
**Pull request by adlerhurst**

### CODE REVIEW

File: internal/eventstore/locker/lock_test.go [hunk 1]
```diff
@@ -1,125 +1,135 @@
...
-			if err := Renew(tt.fields.db.db, tt.args.tableName, tt.args.lockerID, tt.args.viewModel, tt.args.waitTime); (err != nil) != tt.wantErr {
-				t.Errorf("locker.Renew() error = %v, wantErr %v", err, tt.wantErr)
-			}
-			if err := tt.fields.db.mock.ExpectationsWereMet(); err != nil {
-				t.Errorf("not all database expectations met: %v", err)
-			}
-		})
-	}
-}
+// import (
```
Comment (fgerschwiler) [inline internal/eventstore/locker/lock_test.go:3]: Uncommentet Test?

File: internal/management/repository/eventsourcing/eventstore/user.go [hunk 2]
```diff
@@ -184,10 +184,11 @@ func (repo *UserRepo) UserChanges(ctx context.Context, id string, lastSequence u
 	}
 	for _, change := range changes.Changes {
 		change.ModifierName = change.ModifierID
+		// user, _ := repo.View.UserByID(change.ModifierID)
```
Comment (fgerschwiler) [inline internal/management/repository/eventsourcing/eventstore/user.go:187]: removif not used uncommented code?

File: internal/eventstore/spooler/spooler_test.go [hunk 3]
```diff
@@ -1,449 +1,447 @@
 package spooler
 
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
+// import (
```
Comment (fgerschwiler) [inline internal/eventstore/spooler/spooler_test.go:3]: uncommented code?

**Comment by adlerhurst** [comment]
duplicate of https://github.com/caos/zitadel/pull/1110

**Pull request closed without merging**